wdl-engine/tests/tasks/test-join-paths/source.wdl

```diff
@@ -1,15 +1,13 @@
 version 1.2
 
 task resolve_paths_task {
-  input {
-    File abs_file = "/usr"
-    String abs_str = "/usr"
-    String rel_dir_str = "bin"
-    File rel_file = "echo"
-    File rel_dir_file = "mydir"
-    String rel_str = "mydata.txt"
-  }
-
+  File abs_file = "/usr"
```

Comment:
This test didn't need these to be inputs, which would require these paths to exist; changed them to private declarations.